posthog/session_recordings/queries/test/listing_recordings/test_session_recordings_list_by_cohort.py

```diff
@@ -0,0 +1,511 @@
+                                "type": "events",
+                                "order": 0,
+                                "name": "$pageview",
+                            }
+                        ],
+                    },
+                    [],
+                )
+
+                self._assert_query_matches_session_ids(
+                    {
+                        "properties": [
+                            {
+                                "key": "id",
+                                "value": cohort.pk,
+                                "operator": "in",
+                                "type": "cohort",
+                            }
+                        ],
+                        "events": [
+                            {
+                                "id": "custom_event",
+                                "type": "events",
+                                "order": 0,
+                                "name": "custom_event",
+                            }
+                        ],
+                    },
+                    [session_id_two],
+                )
+
+    @snapshot_clickhouse_queries
+    @also_test_with_materialized_columns(person_properties=["$some_prop"])
+    def test_internal_account_filter_with_cohort_properties(self) -> None:
+        with self.settings(USE_PRECALCULATED_CH_COHORT_PEOPLE=True):
+            with freeze_time("2021-08-21T20:00:00.000Z"):
+                user_one = "test_filter_with_cohort_properties-user"
+                user_two = "test_filter_with_cohort_properties-user2"
+                session_id_one = "session_not_in_cohort"
+                session_id_two = "session_in_cohort"
+
+                Person.objects.create(team=self.team, distinct_ids=[user_one], properties={"email": "bla"})
+                Person.objects.create(
+                    team=self.team,
+                    distinct_ids=[user_two],
+                    properties={"email": "bla2", "$some_prop": "some_val"},
+                )
+                cohort = Cohort.objects.create(
+                    team=self.team,
+                    name="cohort1",
+                    groups=[
+                        {
+                            "properties": [
+                                {
+                                    "key": "$some_prop",
+                                    "value": "some_val",
+                                    "type": "person",
+                                }
+                            ]
+                        }
+                    ],
+                )
+                cohort.calculate_people_ch(pending_version=0)
+
+                self.team.test_account_filters = [
+                    {
+                        "key": "id",
+                        "value": cohort.pk,
+                        "operator": "not_in",
+                        "type": "cohort",
+                    }
+                ]
+                self.team.save()
```

Comment:
this test puts the cohort onto the test account filters instead of using it directly